CHANGELOG.md

```diff
@@ -0,0 +1,109 @@
+# Change Log
+
+## [v1.3.5](https://github.com/* remote origin
```

Comment:
They tagged first and then added an entry to a change log :-) I merged this commit as well. Not much has changed through..